src/pages/patterns/loaders.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: Loaders
+description: >
+  Loaders are used when certain elements or parts of the page could take longer to display.
+---
+
+![Loaders intro](figma:patterns-loaders-intro.png)
+
+## Overview
+
+Different loaders are used to communicate structure and context of the content being loaded,
+each one tailored to a specific use case.
+
+<table>
```

Comment:
if it would be easier for you, you could convert this table into a markdown table and use `[text](link)` for the links. I see you have a 30% width though and I don't know if that could be retained. Either way, just throwing it out there.

Reply:
here's a basic translation of the code into that format as an example:

```
| Component                       | Description                                                               |
| ------------------------------- | ------------------------------------------------------------------------- |
| [Skeleton](/components/spinner) | Default Loader. Used when the content on the pagei s known or predictable |
```

Reply:
I thought of the same thing! This would be an improvement.

Reply:
Just tried removing the 30% from the deploy preview and it seems to render the same on narrow and wide widths. I think its worth doing @steue - let me know if i can help at all.

Reply:
+1. Let's use markdown; this is consistent with other pages such as https://garden.zendesk.com/content/voice-and-tone. Also, please remember to use markdown rather than `<a>` tags for links.

Reply:
nice! i've changed the tables to use markdown. the 30% column width really did help with readability as now without it there's a bit too much space in between the columns.

i did some research and it looks like there isn't a (non hacky) way to do column sizing with markdown? that said, i think its okay for now.